internal/pkg/util/util.go

```diff
@@ -113,3 +113,29 @@ func unescapeByte(input string, i int) (byte, error) {
 	}
 	return 0, fmt.Errorf("invalid input")
 }
+
+func FormatTable(rows [][]string) string {
+	maxValueWidthPerColumn := []int{}
+	for _, row := range rows {
+		for column, value := range row {
+			for len(maxValueWidthPerColumn) <= column {
+				maxValueWidthPerColumn = append(maxValueWidthPerColumn, 0)
+			}
+			if len(value) > maxValueWidthPerColumn[column] {
+				maxValueWidthPerColumn[column] = len(value)
+			}
+		}
+	}
+	sb := &strings.Builder{}
```

Comment:
Actually we can remove the `&` here and add `&` where we pass `sb` to `fmt.Fprintf`, this way we can remove a heap allocation if go isn't smart enough to figure this out